compiler/lineinfos.nim

```diff
@@ -27,135 +27,91 @@ proc createDocLink*(urlSuffix: string): string =
-    warnUnsafeDefault: "The '$1' type doesn't have a valid default value",
-    warnProveInit: "Cannot prove that '$1' is initialized. This will become a compile time error in the future.",
-    warnProveField: "cannot prove that field '$1' is accessible",
-    warnProveIndex: "cannot prove index '$1' is valid",
-    warnUnreachableElse: "unreachable else, all cases are already covered",
-    warnStaticIndexCheck: "$1",
-    warnGcUnsafe: "not GC-safe: '$1'",
-    warnGcUnsafe2: "$1",
-    warnUninit: "use explicit initialization of '$1' for clarity",
-    warnGcMem: "'$1' uses GC'ed memory",
-    warnDestructor: "usage of a type with a destructor in a non destructible context. This will become a compile time error in the future.",
-    warnLockLevel: "$1",
-    warnResultShadowed: "Special variable 'result' is shadowed.",
-    warnInconsistentSpacing: "Number of spaces around '$#' is not consistent",
-    warnCaseTransition: "Potential object case transition, instantiate new object instead",
-    warnCycleCreated: "$1",
-    warnObservableStores: "observable stores to '$1'",
-    warnUser: "$1",
-    hintSuccess: "operation successful: $#",
-    # keep in sync with `testament.isSuccess`
-    hintSuccessX: "${loc} lines; ${sec}s; $mem; $build build; proj: $project; out: $output",
-    hintCC: "CC: $1",
-    hintLineTooLong: "line too long",
-    hintXDeclaredButNotUsed: "'$1' is declared but not used",
-    hintConvToBaseNotNeeded: "conversion to base object is not needed",
-    hintConvFromXtoItselfNotNeeded: "conversion from $1 to itself is pointless",
-    hintExprAlwaysX: "expression evaluates always to '$1'",
-    hintQuitCalled: "quit() called",
-    hintProcessing: "$1",
-    hintCodeBegin: "generated code listing:",
-    hintCodeEnd: "end of listing",
-    hintConf: "used config file '$1'",
-    hintPath: "added path: '$1'",
-    hintConditionAlwaysTrue: "condition is always true: '$1'",
-    hintConditionAlwaysFalse: "condition is always false: '$1'",
-    hintName: "$1",
-    hintPattern: "$1",
-    hintExecuting: "$1",
-    hintLinking: "$1",
-    hintDependency: "$1",
-    hintSource: "$1",
-    hintPerformance: "$1",
-    hintStackTrace: "$1",
-    hintGCStats: "$1",
-    hintGlobalVar: "global variable declared here",
-    hintExpandMacro: "expanded macro: $1",
-    hintUser: "$1",
-    hintUserRaw: "$1",
-    hintExtendedContext: "$1",
-    hintMsgOrigin: "$1",
-  ]
+    errUnknown = "unknown error"
+    errInternal = "internal error: $1"
+    errIllFormedAstX = "illformed AST: $1"
+    errCannotOpenFile = "cannot open '$1'"
+    errXExpected = "'$1' expected"
+    errGridTableNotImplemented = "grid table is not implemented"
+    errGeneralParseError = "general parse error"
+    errNewSectionExpected = "new section expected"
+    errInvalidDirectiveX = "invalid directive: '$1'"
+    errProveInit = "Cannot prove that '$1' is initialized."
+    # deadcode
```

Comment:
indent `deadcode` so it keeps referring to `errProveInit`

Reply:
Good catch!